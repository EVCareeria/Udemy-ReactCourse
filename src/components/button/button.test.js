import Button, {BUTTON_TYPE_CLASSES} from "./button";
import renderer from 'react-test-renderer'



it('button renders correctly', () => {
    const button = renderer
        .create(<Button isLoading={true}></Button>)
    expect(button).toMatchSnapshot();
});