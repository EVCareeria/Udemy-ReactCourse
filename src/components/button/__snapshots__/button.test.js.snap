// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders correctly 1`] = `
<button
  className="sc-bczRLJ bQaacd"
  disabled={true}
>
  <div
    className="sc-hKMtZM klHMhN"
  />
</button>
`;
